tests/unit/metadata/test_multi_table.py

```diff
@@ -613,6 +613,90 @@ def test_add_relationship_child_key_is_primary_key(self):
         with pytest.raises(InvalidMetadataError, match=err_msg):
             metadata.add_relationship('table', 'table2', 'pk', 'pk')
 
+    def test_remove_relationship(self):
+        """Test all relationships are removed using ``remove_relationship``."""
+        # Setup
+        instance = MultiTableMetadata()
+        parent_table = Mock()
+        parent_table.primary_key = 'id'
+        parent_table.columns = {
+            'id': {'sdtype': 'id'},
+            'session': {'sdtype': 'numerical'},
+            'transactions': {'sdtype': 'numerical'},
+        }
+
+        child_table = Mock()
+        child_table.primary_key = 'session_id'
+        child_table.columns = {
+            'user_id': {'sdtype': 'id'},
+            'alternate_id': {'sdtype': 'id'},
+            'session_id': {'sdtype': 'numerical'},
+            'transactions': {'sdtype': 'numerical'},
+        }
+
+        instance.tables = {
+            'users': parent_table,
+            'sessions': child_table,
+        }
+        instance.relationships = [
+            {
+                'parent_table_name': 'users',
+                'child_table_name': 'sessions',
+                'parent_primary_key': 'id',
+                'child_foreign_key': 'user_id',
+            },
+            {
+                'parent_table_name': 'users',
+                'child_table_name': 'sessions',
+                'parent_primary_key': 'id',
+                'child_foreign_key': 'alternate_id',
+            }
```

Comment:
Just for sanity, can we add another relationship that doesn't match the parent and child table to make sure it isn't removed